docs/backend/upgrading/version-specific-migration/upgrade-to-60.md

```diff
@@ -294,4 +294,251 @@ The same change is needed for the no longer supported `includeDependenciesOverri
+
+In Plone 6, when generating a tag for in this case the preview scale, a unique url is generated, and information for this scale is pre-registered.
+Only when the browser requests the scaled image at this url, does Plone generate the scale.
+This avoids generating image scales that never get used.
+
+This performance improvement makes two other image improvements possible.
+These follow in the sections directly below.
+
+Add-on authors do not *have* to change anything.
+But it is a good idea to check how you are using images, otherwise you miss out on this improvement.
+If you call the `tag` method like above, you are good.
+If you use the `scale` method and then use its `tag` method, then you should change:
+
+```diff
+  <img tal:define="images context/@@images"
+-      tal:replace="structure python:images.scale('image', scale='preview').tag()" />
++      tal:replace="structure python:images.tag('image', scale='preview')" />
+```
+
+Alternatively, you can explicitly use the new `pre` argument, but this will fail on Plone 5:
+
+```xml
+<img tal:define="images context/@@images"
+     tal:replace="structure python:images.scale('image', scale='preview', pre=True).tag()" />
+```
+
+```{note}
+There now is an image test page that shows several scales of an image, in various modes.
+In your browser to to an image and add `/@@images-test` to the end of the url.
+```
+
+```{seealso}
+- [plone/plone.scale PR 57](https://github.com/plone/plone.scale/pull/57)
+- [plone/plone.namedfile PR 113](https://github.com/plone/plone.namedfile/pull/113)
+```
+
+
+(v60-responsive-image-support-label)=
+
+## Responsive image support
+
+Responsive image support was added with picture tags.
+For more information, see {ref}`classic-ui-images-responsive-image-support`.
+
+In an add-on nothing needs to be changed.
+But if you want to use the responsive image support, you should use the `picture` method:
+
+```diff
+  <img tal:define="images context/@@images"
+-      tal:replace="structure python:images.tag('image', scale='preview')" />
++      tal:replace="structure python:images.picture('image', picture_variant='small')" />
+```
+
+
+
+(v60-image-scale-catalog-label)=
+
+## Store image scale info in catalog metadata
+
+When you add or edit an image, Plone 6 pre-registers all scales and stores info about them in the portal catalog.
+The catalog brain of an image then has all the needed information about each scale, especially the unique url and the width and height.
+This is used on lists of images to be able show a scale in a tag without waking up the image objects from the database.
+In other words: this speeds up pages that contain lots of images.
+
+Add-on authors do not have to change anything: this happens automatically.
+If you have a very special use case, you can influence this with some new adapters.
+
+```{note}
+When upgrading your Plone Site to Plone 6.0, the inline migration finds all images in your site.
```

Comment:
Not sure whether "inline" or "in-place" is the proper term. I would go with "in-place" because "inline" is used more within code or HTML, and not so much in software in general.

```suggestion
When upgrading your Plone Site to Plone 6.0, the in-place migration finds all images in your site.
```